packages/sku/config/webpack/webpack.config.js

```diff
@@ -58,28 +58,11 @@ const makeWebpackConfig = ({
 
   const vocabOptions = getVocabConfig();
 
-  const envVars = lodash
-    .chain(env)
-    .mapValues((value, key) => {
-      if (typeof value !== 'object') {
-        return JSON.stringify(value);
-      }
-
-      const valueForEnv = value[args.env];
-
-      if (typeof valueForEnv === 'undefined') {
-        console.log(
-          `WARNING: Environment variable "${key}" is missing a value for the "${args.env}" environment`,
-        );
-        process.exit(1);
-      }
-
-      return JSON.stringify(valueForEnv);
-    })
-    .set('SKU_ENV', JSON.stringify(args.env))
-    .set('PORT', JSON.stringify(port))
-    .mapKeys((value, key) => `process.env.${key}`)
-    .value();
+  const envars = stringifyEnvarValues({
```

Comment:
Is the variable rename intentional?

I assumed this was: `Environment Variables => env var => envVar`

Reply:
It was intentional. `envVar` works I guess, but [`envar` is much more common](https://trends.google.com/trends/explore?date=today%205-y&q=envar,envvar&hl=en-AU).